ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -1776,20 +1788,28 @@ impl<'a> Assemble<'a> {
     /// # Arguments
     ///
     /// * `tgt_vars` - The target locations. If `None` use `self.loop_start_locs` instead.
-    fn write_jump_vars(&mut self, tgt_vars: Option<&[VarLocation]>) {
+    fn write_jump_vars(&mut self, iidx: InstIdx, is_sidetrace: bool) {
+        let (tgt_vars, src_ops) = if is_sidetrace {
+            // Side-traces don't have a body.
```

Comment:
This briefly confused me, but I then realised what it's saying. I think we should document *somewhere* this property, because it's kind of fundamental to the things we can do with a side-trace.

Reply:
[6b79d8e](https://github.com/ykjit/yk/pull/1510/commits/6b79d8effdac0f3d4051d7b7ea2f12f98c43c664)